MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,121 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+	// MARK: Все переменные
+	private var questions: [QuizQuestion] = [
+		QuizQuestion(image: "The Godfather", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "The Dark Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "Kill Bill", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "The Avengers", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "Deadpool", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "The Green Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+		QuizQuestion(image: "Old", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+		QuizQuestion(image: "The Ice Age Adventures of Buck Wild", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+		QuizQuestion(image: "Tesla", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+		QuizQuestion(image: "Vivarium", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false)]
+	private var currentQuestionIndex: Int = 0
+	private var correctAnswers: Int = 0
+	private var countOfQuations: Int { questions.count }
+	
+	// MARK: Запуск экрана
     override func viewDidLoad() {
         super.viewDidLoad()
+		showQuiz(quiz: convert(model: questions[0]))
+//		imageView.layer.masksToBounds = true
+//		imageView.layer.borderWidth = 1
+//		imageView.layer.borderColor = UIColor.white.cgColor
+		imageView.layer.cornerRadius = 6
     }
+	// MARK: View модели для состояний
+	
+	// Вью-модель для состояния "Вопрос задан"
+	private struct QuizStepViewModel {
+	  let image: UIImage
+	  let question: String
+	  let questionNumber: String
+	}
+	/// Вью-модель для состояния "Результат квиза"
+	private struct QuizResultsViewModel {
+	  let title: String
+	  let text: String
+	  let buttonText: String
+	}
+	/// структура вопроса
+	private struct QuizQuestion {
+	  let image: String
+	  let text: String
+	  let correctAnswer: Bool
+	}
+	
+	/// функция конвертации вопроса в модель
+	private func convert(model: QuizQuestion) -> QuizStepViewModel {
+		return QuizStepViewModel(image: UIImage(named: model.image) ?? UIImage(),
+								 question: model.text,
+								 questionNumber: "\(1)/\(10)")
```

Comment:
получилось что-то довольно сложное :) давай проще:
```suggestion
								 questionNumber: "1/10")
```